runtime/src/runtime/fvm.rs

```diff
@@ -552,14 +550,13 @@ where
 /// 5a. In case of error, aborts the execution with the emitted exit code, or
 /// 5b. In case of success, stores the return data as a block and returns the latter.
 pub fn trampoline<C: ActorCode>(params: u32) -> u32 {
-    fvm::debug::init_logging();
```

Comment:
If you want to set the max log level to trace, you can do that here. `init_logging` is designed to be unopinionated and just initialize the logging framework.

Reply:
The v2 SDK initialises it to discard messages! As someone not deeply familiar with Rust logging configuration, this was extremely frustrating to discover. Also it still doesn't work, for reasons I don't understand yet.

The v3 SDK does initialise it to tracing. Maybe it'll work, and maybe we'll want to switch to it when we upgrade.

`init_logging` is opinionated about the format of the string that is logged, at least. We may well want to change it.